src/euphorie/client/browser/organisation.py

```diff
@@ -192,6 +200,13 @@ def publishTraverse(self, request, organisation_id):
         request.set(self.organisation_id_key, organisation_id)
         return self
 
+    @property
+    def role_options(self):
+        """Return a list of options for the role field."""
+        options = deepcopy(self._known_roles)
+        options[0]["checked"] = "checked"
+        return options
```

Comment:
Is it needed to have this property, AFAICS you can just add `checked: "checked"` in line 44

Reply:
No, that will mess up editing existing members (line 746).